data_steward/analytics/cdr_ops/retraction_validation.py

```diff
@@ -97,19 +97,19 @@
 table_row_counts_query = JINJA_ENV.from_string('''
 SELECT
 \'{{table_name}}\' as table_name,
-q4r3_minus_aian_row_count,
-q4r4_row_count,
-case when q4r3_minus_aian_row_count = q4r4_row_count then 'OK'
-  when q4r3_minus_aian_row_count is null AND q4r4_row_count is null then 'OK'
-  when q4r3_minus_aian_row_count is NOT NULL and q4r4_row_count is null then 'PROBLEM'
-  when q4r3_minus_aian_row_count is NULL and q4r4_row_count is not NULL then 'PROBLEM'
+newr3_minus_aian_row_count,
```

Comment:
would this be better as `old_minus_aian_row_count` so we don't have `r3` and `r4` still mixed into the naming scheme?